community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -838,6 +838,13 @@ public String providerName()
     public static final Setting<String> default_advertised_address =
             setting( "dbms.connectors.default_advertised_address", STRING, "localhost" );
 
+    @Description( "Value of the Access-Control-Allow-Origin header sent over any HTTP or HTTPS " +
+            "connector. This defaults to '*', which allows broadest compatibility but is " +
+            "least secure. Note that any URI provided here limits HTTP/HTTPS access to " +
```

Comment:
Tiny sidenote: I'm not sure if "least secure" is helpful; it might surprise users to see this say that the default is whatever is the least secure option. 

AFAIK, there is no security issue with CORS on the server right, since none of our APIs accepts credentials in the form of cookies, cross-site request forgery isn't possible. 

Meaning: There is no security difference between turning CORS on or off, for our particular API, I don't think?

Reply:
This is chiefly for the browser.